week2/vite-project/src/pages/todo.js

```diff
@@ -104,15 +104,15 @@ function Todo($container) {
 
   const storedCategoryOrder = JSON.parse(localStorage.getItem("categoryOrder"));
 
-  const sortedTodoInfo = todoInfo.sort((a, b) => {
-    return (
-      storedCategoryOrder.indexOf(a.category) -
-      storedCategoryOrder.indexOf(b.category)
-    );
-  });
-
   const currentTodoOrder =
-    storedCategoryOrder === null ? todoInfo : sortedTodoInfo;
+    storedCategoryOrder === null
+      ? todoInfo
+      : todoInfo.sort((a, b) => {
+          return (
+            storedCategoryOrder.indexOf(a.category) -
+            storedCategoryOrder.indexOf(b.category)
+          );
+        });
```

Comment:
```suggestion
storedCategoryOrder 
      ?  todoInfo.sort((firstTodo, secondTodo) => {
          return (
            storedCategoryOrder.indexOf(firstTodo.category) -
            storedCategoryOrder.indexOf(secondTodo.category)
          );
        }); 
        : todoInfo;
```
null이랑 비교하는 이유가뭘까요?! 있고 없고로 하면 안데에?!